webapp/src/app/pages/landing-page/landing-page.component.ts

```
@@ -84,19 +84,31 @@
+
+         
         this.courses = courses;
 console.log('Courses:', this.courses);
         for (var course of this.courses) {
+          try{
+
           this.Users = [];
 
           this.Users = course.users;
-          console.log(' this.Userss: ', this.Users); 
           let userModerator = this.Users.find(
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'userModerator' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)